ts/cognitarium-schema/package.json

```diff
@@ -28,5 +28,8 @@
         "clean": "rm -rf gen-ts && mkdir gen-ts",
         "build": "tsc"
     },
-    "exports": "./dist/schema.d.ts"
+    "exports": "./dist/schema.d.ts",
+    "typesVersions": {
```

Comment:
Why not using the `types` property? 
According to the docs: _Set the types property to point to your bundled declaration file_

Reply:
Yes, let's try this ! It seems better.